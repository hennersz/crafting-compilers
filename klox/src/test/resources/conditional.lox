print "hi" or 2;
print nil or "yes";
print true and 3 or 4;