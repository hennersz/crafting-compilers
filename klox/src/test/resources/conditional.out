hi
yes
3.0